config/default.yml

```diff
@@ -23,6 +23,9 @@ RSpec/ExampleWording:
     not: does not
   IgnoredWords: []
 
+RSpec/BeforeAfterAll:
+  Description: 'Avoid the use of before/after(:all) as they are not rolled back and leads to flakiness.'
```

Comment:
"and may lead" ?

Also, "flakiness" feels a little vague – how about something like "and may lead to database state leaking between examples"?